prompterator/main.py

```diff
@@ -470,6 +470,30 @@ def set_up_prompt_attrs_area(st_container):
             height=c.DATA_POINT_TEXT_AREA_HEIGHT,
         )
 
+        system_prompt = env.from_string(st.session_state.system_prompt).render(
```

Comment:
@samsucik @sweco this is the very same thing as in:

https://github.com/slidoapp/prompterator/blob/280406f528c613cf23a86f8cc57f2243c60f834c/prompterator/utils.py#L228-L229

I guess if we like it, we could abstract it out somewhere